### DISCUSSION TITLE
how to show only a number in metrics

### DISCUSSION THREAD

**Discussion by Jedore** [apache/skywalking#9331] [Q&A] [answered]
SkyWalking version: 9.0.0
Now visualization(chart type) has followed types:
<img width="621" alt="截屏2022-07-11 16 39 38" src="https://user-images.githubusercontent.com/17944025/178223957-95d7a2aa-adfb-4890-b5c0-b811103f4f34.png">
then in line chart every y-axis number relate a specific x-axis time point. But I want another style, only show a total number in a specific period, such as last 24 hours, last week, 30days. 
Just like
<img width="448" alt="Screenshot at Jul 11 16-49-30" src="https://user-images.githubusercontent.com/17944025/178225683-857d9d61-ddf7-4d81-b5d4-df7e67a4e17c.png">
Does skywalking support feature like this? Or what other solution can i use to implement similar functionality?
Thanks!

**Comment by wu-sheng** [comment, marked as answer]
I think we at least don't have this widget. If you want to support this, you have to implement a new widget.

**Reply by Jedore** [reply]
Okay, thanks for your answer.
